packages/path_provider/path_provider_linux/example/linux/flutter/CMakeLists.txt

```diff
@@ -80,7 +80,8 @@ add_custom_command(
   COMMAND ${CMAKE_COMMAND} -E env
     ${FLUTTER_TOOL_ENVIRONMENT}
     "${FLUTTER_ROOT}/packages/flutter_tools/bin/tool_backend.sh"
-      linux-x64 ${CMAKE_BUILD_TYPE}
+      ${FLUTTER_TARGET_PLATFORM} ${CMAKE_BUILD_TYPE}
```

Comment:
This isn't valid on the stable channel.

Reply:
@robert-ancell Ping; is there a specific issue this was trying to solve? This should auto-migrate for people when building on master.

Reply:
This PR was just to test if the CI build failures were due to https://github.com/flutter/plugins/pull/3077. I thought this change was trivial but apparently not trivial enough :)